src/Parallel/Main.hpp

```diff
@@ -103,6 +111,10 @@ class Main : public CBase_Main<Metavariables> {
           reduction_data) noexcept;
 
  private:
+  // Returns the dir name for the next Charm++ checkpoint; checks and errors if
```

Comment:
`next` in the comment is at adds with the function name, no?

Reply:
Oops :(